contracts/Comptroller/Diamond/facets/PolicyFacet.sol

```diff
@@ -353,6 +384,7 @@ contract PolicyFacet is IPolicyFacet, XVSRewardsHelper {
         address borrower,
         uint256 seizeTokens // solhint-disable-line no-unused-vars
     ) external {
+        ensureUserAllowed(liquidator);
```

Comment:
I think this is not needed, because we are already checking the permission for seizing in the `seizeAllowed` check (the "pre-hook")